kitsune/products/views.py

```diff
@@ -44,12 +54,16 @@ def product_landing(request, template, slug):
         else:
             latest_version = 0
 
+    show_community_support = request.LANGUAGE_CODE == settings.LANGUAGE_CODE or is_localized(
```

Comment:
What is ``settings.LANGUAGE_CODE`` ? I think this should be ``settings.WIKI_WIKI_DEFAULT_LANGUAGE``